src/Components/Blueprints/BlueprintsSideBar.tsx

```diff
@@ -55,8 +55,8 @@ type emptyBlueprintStateProps = {
 const BlueprintsSidebar = () => {
   const selectedBlueprintId = useAppSelector(selectSelectedBlueprintId);
   const blueprintSearchInput = useAppSelector(selectBlueprintSearchInput);
-  const blueprintsOffset = useAppSelector(selectOffset);
-  const blueprintsLimit = useAppSelector(selectLimit);
+  const blueprintsOffset = useAppSelector(selectOffset) || 0;
+  const blueprintsLimit = useAppSelector(selectLimit) || 10;
```

Comment:
Maybe we put these in `constants.ts`?

Reply:
Also maybe put a comment explaining that they're for Cockpit. Future devs might see image-builder API has defaults and think they aren't needed.